src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/RequestApproved.cshtml

```diff
@@ -36,14 +36,33 @@
         </table>
 
         <h2 class="heading-medium">What happens next?</h2>
-        <p id="changeMainCopy">
-            Your cohort has been approved. To update the details for any apprentices in this cohort you will need to 
-            <a href="@Url.Action("Index" ,"ManageApprentices")">manage your apprentices</a>.
-        </p>
-
-        @if (!string.IsNullOrEmpty(Model.RedirectUrl))
+            
+        @if (Model.IsTransfer)
+        {
+            <ul class="list list-bullet text">
+                <li>
+                    A transfer request has been sent to the employer who is funding this cohort
+                </li>
+                <li>
+                    Once the transfer request is approved, you'll be able to <a href="@Url.Action("Index", "ManageApprentices")">view and manage the apprentices</a>
+                </li>
+            </ul>
+        }
+        else
         {
-            <a href="@Model.RedirectUrl" aria-label="@Model.RedirectLinkText">@Model.RedirectLinkText</a>
+            <p id="changeMainCopy">
+                Your cohort has been approved. To update the details for any apprentices in this cohort you will need to
+                <a href="@Url.Action("Index", "ManageApprentices")">manage your apprentices</a>.
+            </p>
+        }
+        
+        @if (Model.HasOtherCohortsAwaitingApproval)
+        {
+            <a href="@Url.Action("ReadyForReview")">Return to Approve cohorts</a>
```

Comment:
As discussed, these Url's when generated in CommitmentController used to supply the provider id (and the actions seem to require them). Do we need to supply the provider id here?

Also, is the Url.Action clever enough to pick up the action is in the commitment controller (now that the Url generation has moved into the view).

Reply:
Chris has shown this works. Looks like Url.Action uses the route values from the current request...
https://stackoverflow.com/questions/7133223/asp-net-mvc-url-action-adds-current-route-values-to-generated-url